ex1.py

```diff
@@ -0,0 +1,4 @@
+n = 1439  #сюда вписать любое число от 0 до 1439
+b = n // 60
+c = n - b*60
+print(b, ':', c, )    #мне интересно, как сделать чтоб между числами в ответе не было пробелов
```

Comment:
вот так не будет пробелов
```suggestion
print(f'{b}:{c}')
```